tests/FSharp.Data.GraphQL.IntegrationTests.Server/FSharp.Data.GraphQL.IntegrationTests.Server.fsproj

```diff
@@ -7,17 +7,20 @@
 
   <ItemGroup>
     <PackageReference Include="Giraffe" />
+    <PackageReference Include="Newtonsoft.Json" />
```

Comment:
Why do we need this? Do we use it anywhere? I thought I removed that
It would be nice to remove that if it exists anywhere.
Ah, I suppose it is in WebSockets, right?

Reply:
This is from the old IntegrationTests.Server code, yes.  I removed it on 2023/02/LHK-607-Introspection_Query .